app/controllers/publishers/vacancies/base_controller.rb

```diff
@@ -3,7 +3,7 @@
 class Publishers::Vacancies::BaseController < Publishers::BaseController
   include Publishers::Wizardable
 
-  helper_method :process_steps, :step_current, :steps_adjust, :vacancy
+  helper_method :process_steps, :step_current, :steps_adjust, :steps_config, :vacancy
```

Comment:
I can't see where this new helper_method gets used. Is it for the methods you added to VacanciesHelper?

Reply:
Yes!

Reply:
I think it makes more sense to pass in `steps_config` as an argument to the VacanciesHelper methods (similar to the other helper methods), because then (a) it's clear what info the helpers depend on, and (b) the vacancies helper module should be able to slot in anywhere (in other words, it should be 'modular'), without dragging along a hidden dependency. One hidden dependency is tolerable, but if it gets more over time it will become 'spaghetti code' that's difficult to change without causing more changes elsewhere.